src/models.jl

```diff
@@ -42,21 +71,26 @@ mutable struct Deterministic{T} <: ModelInput
             throw(DimensionMismatch("exogenous_forecast timestamps must be the same."))
         end
         # Test if all the exogenous forecast timestamps has length equal to steps ahead
-        if has_exogenous_variable && !assert_length_time_series_timestamp(exogenous_forecast[1], parameters["steps_ahead"])
-            throw(ErrorException("exogenous_forecast timestamps must have the same length as steps ahead."))
-        end
+        # if has_exogenous_variable && !assert_length_time_series_timestamp(exogenous_forecast[1], parameters["steps_ahead"])
+        #     throw(ErrorException("exogenous_forecast timestamps must have the same length as steps ahead."))
+        # end
```

Comment:
You should either remove it or let it there, comment some piece of code is not an option